images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -45,6 +45,7 @@ RUN apt-get update \
         libssl1.0.2 \
         libstdc++6 \
         zlib1g \
+        libgdiplus \
```

Comment:
Adding this library to 'build' images is not required for AppService scenarios for example, but this is for VSO users who might want to run an app in the build image.